webpack/assets/javascripts/react_app/redux/actions/hosts/powerStatus/__snapshots__/powerStatus.test.js.snap

```diff
@@ -3,8 +3,11 @@
 exports[`hosts actions creates HOST_POWER_STATUS_REQUEST and fails when host not found 1`] = `
```

Comment:
Do these snaps need to be removed from this PR now that PowerStatus will be in the second PR?

Reply:
no, this changes are made by this PR